src/imports/components/Footer/index.styl

```diff
@@ -0,0 +1,17 @@
+@import "~styles/config.styl"
+
+.footer
+  display flex
+  justify-content space-between
+  max-width 1432px
+  margin 0 auto 26px
```

Comment:
Change to margin-bottom 26px

Reply:
I totally agree with you about using margin-{side} css property type when we need to make one side margin. This case requires two indents at least.